src/service/layers-impl.js

```diff
@@ -242,12 +265,15 @@ export class LayoutLayers {
    * #buildCallback, #layoutCallback, viewport changes size).
    *
    * @param {!Element} element A regular or AMP Element
+   * @param {boolean=} opt_force
    */
-  remeasure(element) {
-    const layer = LayoutElement.getParentLayer(element);
-    if (layer) {
-      layer.remeasure();
+  remeasure(element, opt_force) {
+    const layout = this.add(element);
+    const from = layout.getParentLayer() || layout;
+    if (opt_force) {
+      from.requestRemeasure();
     }
+    from.remeasure();
```

Comment:
Feels like this should be inverted name-wise. 

```
if (force)
  remeasure()
else
  requestRemeasure()
```

Reply:
Ok, how about changing `requestMeasure` to `setDirty` or `setNeedsRemeasure`? 🚲 🏚

Reply:
Sure.